Missing image manila-provisioner during uploading images in Makefile

<!-- This form is for bug reports and feature requests! -->

**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
/kind bug
> /kind feature


**What happened**:
upload image manila-provisioner failed when running Makefile, here is my output:
2018-06-24 01:54:44.993212 | ubuntu-xenial-vexxhost | docker push docker.io/k8scloudprovider/manila-provisioner:latest
2018-06-24 01:54:45.088064 | ubuntu-xenial-vexxhost | The push refers to a repository [docker.io/k8scloudprovider/manila-provisioner]
2018-06-24 01:54:45.088299 | ubuntu-xenial-vexxhost | An image does not exist locally with the tag: k8scloudprovider/manila-provisioner
2018-06-24 01:54:45.099917 | ubuntu-xenial-vexxhost | Makefile:254: recipe for target 'upload-images' failed
2018-06-24 01:54:45.100076 | ubuntu-xenial-vexxhost | make: *** [upload-images] Error 1
2018-06-24 01:54:45.379174 | ubuntu-xenial-vexxhost | ERROR
2018-06-24 01:54:45.379500 | ubuntu-xenial-vexxhost | {
2018-06-24 01:54:45.379589 | ubuntu-xenial-vexxhost |   "delta": "0:02:29.734363",
2018-06-24 01:54:45.379663 | ubuntu-xenial-vexxhost |   "end": "2018-06-24 01:54:45.107716",
2018-06-24 01:54:45.379730 | ubuntu-xenial-vexxhost |   "failed": true,
2018-06-24 01:54:45.379794 | ubuntu-xenial-vexxhost |   "rc": 2,
2018-06-24 01:54:45.379926 | ubuntu-xenial-vexxhost |   "start": "2018-06-24 01:52:15.373353"
2018-06-24 01:54:45.379998 | ubuntu-xenial-vexxhost | }

[1] http://logs.openlabtesting.org/logs/periodic/github.com/kubernetes/cloud-provider-openstack/master/cloud-provider-openstack-acceptance-test-e2e-conformance-stable-branch-v1.11/d5bb635/job-output.txt.gz

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:


**Anything else we need to know?**:

**Environment**:
- openstack-cloud-controller-manager version:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

--- a/Makefile
+++ b/Makefile
@@ -58,7 +58,7 @@ endif
 depend-update: work
 	dep ensure -update -v
 
-build: openstack-cloud-controller-manager cinder-provisioner cinder-flex-volume-driver cinder-csi-plugin k8s-keystone-auth client-keystone-auth octavia-ingress-controller
+build: openstack-cloud-controller-manager cinder-provisioner cinder-flex-volume-driver cinder-csi-plugin k8s-keystone-auth client-keystone-auth octavia-ingress-controller manila-provisioner
 
 openstack-cloud-controller-manager: depend $(SOURCES)
 	CGO_ENABLED=0 GOOS=$(GOOS) go build \
@@ -185,7 +185,7 @@ realclean: clean
 shell:
 	$(SHELL) -i
 
-images: image-controller-manager image-flex-volume-driver image-provisioner image-csi-plugin image-k8s-keystone-auth image-octavia-ingress-controller
+images: image-controller-manager image-flex-volume-driver image-provisioner image-csi-plugin image-k8s-keystone-auth image-octavia-ingress-controller image-manila-provisioner
 
 image-controller-manager: depend openstack-cloud-controller-manager
 ifeq ($(GOOS),linux)

